www/src/templates/template-docs-local-packages.js

```diff
@@ -69,6 +70,11 @@ class DocsLocalPackagesTemplate extends React.Component {
                 : npmPackageNotFound.lastPublisher
             }
           />
+          <Unbird
+            dataSetId="5c1ac24b4a828a169b6c235c"
```

Comment:
do we want to have these values hardcoded or do we want to put this in .env folder and consume it?

Reply:
it was more a suggestion, I would be cool with keeping it but I love the change!

Reply:
it was a good idea, and itched the back of my mind when I did this originally :)